packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx

```diff
@@ -3,4 +3,86 @@ id: 07-ui-customization-options
 title: UI Customization Options
 ---
 
-Stay tuned for more information on this topic.
+:::warning
+Here is a list of things still to address on this page:
+
+- Show an example of how to use one of the pre-built components
+- Add more CSS theming examples
+
+(Whenever you add one of the sections please remove the point from the list.)
+:::
+
+## Introduction
+
+There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in video components ready for use.
+In addition, it allows for styling and custom theming options with custom CSS.
+
+For custom use cases it is also possible to only use the low-level client to provide completely custom UI to fully tailor the SDK towards your needs.
+
+This page guides you through all of them with increasing levels of customization.
+
+### Using pre-built components
+
+The SDK comes with a lot of pre-built components for the different elements of the UI.
+
+:::tip
+For a list of all available UI components head over to the [Components](../../ui/01-ui-overview) section.
+:::
+
+### Theming with CSS
+
+If you only want to customize the styling of the available components, there is a wide range of CSS variables that you can override.
+With that, you can achieve a significant amount of theming in your video product.
+
+The way this works is that there are various reserved variable names inside of our [styling package](TODO: link to styling package).
+To change them you can simply override them with your custom values.
+
+Here are a few examples:
+
+```scss
+// Change the font family (make sure to provide enough fallback fonts, though)
+--str-video__font-family: Roboto, sans-serif;
+
+// The primary color that is used
+--str-video__primary-color: green;
+```
+
+There are too many variables that you can change, so here are links to the files that contain them for you to discover:
+
+- [Global Theming](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_global-theme-variables.scss)
+- [Global Layout](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_global-layout-variables.scss)
+- [Color Palette](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_palette-variables.scss)
+- [Icons and Colors](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_variables.scss)
+
+### Using low-level client combined with custom UI
+
+If you want to fully customize the experience, you can also use the client SDK to access the logic and data from the Video SDK and fully customize the UI.
+
+Here is an example of how you can use the `useParticipants()` hook to get all call participants and render them in your application.
+
+```tsx
+import { useParticipants } from '@stream-io/video-react-bindings';
```

Comment:
```suggestion
import { useParticipants } from '@stream-io/video-react-sdk';
```

I'd primarily advise to use the SDK-provided hooks.

Reply:
Corrected the import